src/Refactoring-Missing/ChangesBrowserPresenter.class.st

```diff
@@ -0,0 +1,197 @@
+"
+I am a GUI used to preview the impact of a refactoring and select the changes to apply.
+
+"
+Class {
+	#name : #ChangesBrowserPresenter,
+	#superclass : #ComposablePresenter,
+	#instVars : [
+		'textArea',
+		'changesTree',
+		'changes',
+		'selectedChanges'
+	],
+	#category : #'Refactoring-Missing'
```

Comment:
Why package is named Refactoring-Missing?

Reply:
It was a provisional name, since I planned to add new refactorings in it, I will modify it in the smaller PRs.